webapp/src/license.tsx

```diff
@@ -5,27 +5,25 @@ import {useSelector} from 'react-redux';
 
 import {GlobalState} from '@mattermost/types/store';
 
-const e10 = 'E10';
-const e20 = 'E20';
 const professional = 'professional';
 const enterprise = 'enterprise';
+const enterpriseAdvanced = 'advanced';
 
 // isValidSkuShortName returns whether the SKU short name is one of the known strings;
-// namely: E10 or professional, or E20 or enterprise
+// namely: professional, enterprise or enterprise advanced.
 const isValidSkuShortName = (license: Record<string, string>) => {
     switch (license?.SkuShortName) {
-    case e10:
```

Comment:
Does this break backward compatibility? I don't know what is the latest version that we support from Mattermost or the latest license that we support from Mattermost.